RoFILib/util/topology2dot.cpp

```diff
@@ -0,0 +1,98 @@
+#include <Configuration.h>
+#include <IO.h>
+#include <iostream>
+#include <fstream>
+#include <string>
+
+
+std::string sTS(Side s) {
+    switch(s){
+        case Side::A:
+            return "A";
+        default:
+            return "B";
+    }
+}
+
+std::string dTS(Dock d) {
+    switch(d) {
+        case Dock::Xp:
+            return "X+";
+        case Dock::Xn:
+            return "X-";
+        default:
+            return "Z";
+    }
+}
+
+std::string edgeToDot(const Edge &edge) {
+    ID id1, id2;
+    Side side1, side2;
+    Dock dock1, dock2;
+    if (edge.id1() < edge.id2()) {
+        id1 = edge.id1();
+        id2 = edge.id2();
+        side1 = edge.side1();
+        side2 = edge.side2();
+        dock1 = edge.dock1();
+        dock2 = edge.dock2();
+    } else {
+        id1 = edge.id2();
+        id2 = edge.id1();
+        side1 = edge.side2();
+        side2 = edge.side1();
+        dock1 = edge.dock2();
+        dock2 = edge.dock1();
+    }
+    std::string label = " [label=\"" + sTS(side1) + dTS(dock1) + std::to_string(edge.ori()) + dTS(dock2) + sTS(side2) + "\"]";
+    return "\t" + std::to_string(id1) + " -- " + std::to_string(id2) + label + ";\n";
+}
+
+
+bool drawDot(std::ofstream &output, const Configuration &cfg) {
+    std::string dotGraph;
+
+    dotGraph += "strict graph configuration {\n";
+
+    for(const auto& [id, eList] : cfg.getEdges())
+        for(const auto& mEdge : eList)
+            if(mEdge.has_value())
+                dotGraph += edgeToDot(mEdge.value());
+    
+    dotGraph += "}\n";
+    output << dotGraph;
+    return true;
+}
+
+
+int main(int argc, char* argv[]) {
+    Configuration config;
+
+    if (argc != 3) {
+        std::cout << "Tool expected exactly two args" << std::endl;
```

Comment:
Prosím vypsat help, který říká, jak se to používá - dva argumenty jsou nic neříkající.